docs/webui.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Gravitino web UI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+
+import Image from '@theme/IdealImage'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+This document primarily outlines how users can manage metadata within Gravitino using the web UI, the graphical interface accessible through a web browser as an alterative to writing code or using the REST interface.
```

Comment:
"the graphical interface is accessible..."

Reply:
DONE